### ISSUE TITLE
SelectMultiple - focus order 2.4.3

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet#7671]
### Is this something you or your team is interested in contributing a pull request towards?

None

### Expected Behavior

When I tab into the list of options in the drop the first option should receive focus

### Actual Behavior

The first option does not receive focus until down arrow is pressed

### URL, Screenshot, or CodeSandbox

https://storybook.grommet.io/?path=/story/input-selectmultiple-default--default

### Steps to Reproduce

1. Open the Select multiple drop down
2. Tab to the options
3. First option doesn't receive focus

**Comment by jcfilben** [comment]
This issue is related to SelectMultiple wcag rule 2.4.3, currently on DS site we are incorrectly marking this rule as passed
